lib/mac/src/noble_mac.mm

```diff
@@ -141,12 +141,18 @@
 // write(deviceUuid, serviceUuid, characteristicUuid, data, withoutResponse)
 Napi::Value NobleMac::Write(const Napi::CallbackInfo& info) {
     CHECK_MANAGER()
-    ARG5(String, String, String, Buffer, Boolean)
```

Comment:
isnt it preferable to add comments like :

    ...  /* , Boolean */)

Reply:
@rzr Yeaj, that's a good idea